app/Filament/Resources/AchievementSetResource/RelationManagers/GameHashesRelationManager.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources\AchievementSetResource\RelationManagers;
+
+use App\Models\AchievementSet;
+use App\Models\GameHash;
+use Filament\Forms;
+use Filament\Forms\Components\Select;
+use Filament\Forms\Form;
+use Filament\Notifications\Notification;
+use Filament\Resources\RelationManagers\RelationManager;
+use Filament\Support\Enums\FontFamily;
+use Filament\Tables;
+use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\HtmlString;
+
+class GameHashesRelationManager extends RelationManager
+{
+    protected static string $relationship = 'gameHashes';
+
+    protected static ?string $title = 'Incompatible Hashes';
```

Comment:
As best as I can tell, this implementation assumes all entries in the `gameHashes` list are marked as incompatible. Should the property be renamed to reflect that? `incompatibleGameHashes`?

Are there plans to use the `compatible` field in the table for anything else? Maybe it can be removed and the table renamed to `achievement_set_incompatible_game_hashes`?

Reply:
To the best of my knowledge, there are no plans to use the `compatible` field for anything else. When the table was created, I was a bit less sure, but now that the system design for multiset has started to take a more solid shape, it's easier to see that field is very likely redundant.

I've added a migration to latest which renames the table and drops that field.